2020/02/08 05:31:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:35:00 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:45:02 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:49:24 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 05:55:08 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 06:02:48 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:26:03 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:34:57 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 09:35:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:16:27 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:19:11 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:20:26 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:35:30 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:35:39 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 10:39:18 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:02:04 [warn] 1#1: conflicting server name "localhost" on 0.0.0.0:80, ignored
2020/02/08 11:02:12 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.29.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
2020/02/08 11:02:18 [crit] 7#7: *1 connect() to unix:///app/tmp/sockets/puma.sock failed (2: No such file or directory) while connecting to upstream, client: 172.29.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://unix:///app/tmp/sockets/puma.sock:/", host: "localhost"
